testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksCreateTableAsSelectCompatibility.java

```diff
@@ -226,7 +227,8 @@ public void testReplaceTableWithSchemaChange()
         }
     }
 
-    @Test(groups = {DELTA_LAKE_DATABRICKS, PROFILE_SPECIFIC_TESTS})
+    // Databricks 11.3 doesn't create a checkpoint file at 'CREATE OR REPLACE TABLE' statement
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_EXCLUDE_113, PROFILE_SPECIFIC_TESTS})
```

Comment:
But that doesn't mean we should exclude the test, do it?

Reply:
What we can do in 11.3 is verifying a checkpoint file wasn't created in `CREATE OR REPLACE TABLE` and skip, but I don't feel such logic is helpful in this test method. The test objective is accomplished by other versions. If we verify Databricks behavior (which operation creates a checkpoint file or not), adding dedicated tests look better to me.

Reply:
Can CREATE OR REPLACE TABLE create a checkpoint in 11.3?
Can we force it to do that?

Note that the test specificially wants to test the scenario when checkpoint changes the schema

Reply:
Unfortunately, I couldn't find a way to create a checkpoint using the statement, such options,  alternative procedures or something in 11.3.